### ISSUE TITLE
Location Details - Add distance to Info screen

### ISSUE THREAD

**Issue by RyanTG** [bpoore/pbm-react#34]
I couldn't pull this off myself...

**Comment by bpoore** [comment]
6dc84d3060f09dbfe4e3b9511fbc11e4c81375a1
This information isn't explicitly provided by the API. We've gone back and forth about whether the app will do the distance calculation or have that information provided by the API. While looking into adding in functionality to have the app provide the calculation, I found using the distance value provided by the API for our uses wasn't properly providing the distance for us. I provided a quick fix for this, but should come up with a more robust solution. The problem is that the distance determination coming from the API was using the coordinates for the center of the map rather than the user's location. Let me know what you think about this new change- before I spend time cleaning up the LocationList, we might want to circle back to make sure we're displaying the locations in that list that we want to be.

TL/DR - LocationList was not properly calculating distance for the user, it's fixed now. Do we like showing the Locations visible in the map for that list? That's what's currently happening.

**Comment by RyanTG** [comment]
Seems great! Thanks.

Side question: If I wanted to add an HR tag (or the closest react native equivalent, which is a bottom border) only if the Operator field -OR- the Location Type field -OR- the Location Description field is present, how would I do that?

This doesn't work:
````
                            {location.location_type_id || location.operator_id || location.description &&
                                <View
                                    style={{
                                        borderBottomColor: 'black',
                                        borderBottomWidth: 1,
                                    }}
                                />
                            }
````
If I replace the `II`s with `&&`s then it works if all three are present. But I want OR. I tried to put `II` in all three spots and that didn't cut it.
